src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/TestHelpers.java

```diff
@@ -130,19 +132,21 @@ public static Court createCourt() {
     public static Submission buildSubmission() {
         return Submission
                 .builder()
-                .filingPackage(createPackage(createCourt(), createDocumentList(), createPartyList()))
+                .filingPackage(createPackage(createCourt(), createDocumentList(), createPartyList(), createOrganizationList()))
                 .navigationUrls(createNavigation(SUCCESS_URL, CANCEL_URL, ERROR_URL))
                 .create();
     }
 
     public static FilingPackage createPackage(
             Court court,
             List<Document> documents,
-            List<Party> parties) {
+            List<Individual> parties,
+            List<Organization> organizations) {
         return FilingPackage.builder()
                 .court(court)
                 .documents(documents)
                 .parties(parties)
```

Comment:
This all looks really good.  My only comment is regarding the refactoring that split Parties into Individuals and Organizations - there could be more work here if you deemed appropriate.  You may want to consider keeping the Party term to the interface and use Individual and Organization as the impls. 
i.e, 
line 143 could be `List<Individual> individuals`
line 148 could be `.individuals(individuals)`

Reply:
I agree the reason I left the parties name was is the naming we have to use in the swagger. So I left it named that we throughout. Same as below comment.